src/Client/Pages/Identity/Roles/RolePermissions.razor

```diff
@@ -0,0 +1,50 @@
+﻿@page "/role/{Id}/permissions"
+@attribute [Authorize(Policy = FSHPermissions.Roles.View)]
+@inject IStringLocalizer<RolePermissions> _localizer
+
+<FshTitle Title="@Title" Description="@Description" />
+
+@if (!_loaded)
+{
+    <MudProgressCircular Color="Color.Primary" Indeterminate="true" />
+}
+else
+{
+    <MudTable Hover="true" Elevation="25" Items="RolePermissionsList" Dense="@_dense" Bordered="@_bordered" Striped="@_striped" Filter="new Func<PermissionUpdateDto, bool>(Search)" @bind-permission="_permission">
```

Comment:
@bind-permission, what does this do?
this _permission object isn't used anywhere else...

Reply:
I have to verify , i used as base  the user_role page

Reply:
Yeah I don't think it's necessary (together with the _permission object)

Reply:
Done